Tests/SentryTests/SentryCrash/SentryStacktraceBuilderTests.swift

```diff
@@ -103,14 +100,13 @@ class SentryStacktraceBuilderTests: XCTestCase {
         let filteredFrames = actual.frames.filter { frame in
             return frame.function?.contains("testAsyncStacktraces") ?? false ||
             frame.function?.contains("asyncFrame1") ?? false ||
-            frame.function?.contains("asyncFrame2") ?? false ||
-            frame.function?.contains("asyncAssertion") ?? false
```

Comment:
But why we used to expected to have an asyncAssertion and now we dont?

Reply:
I don't know, but I do know that `asyncAssertion` is never part of the stack traces, and removing this check is better than the test being completely disabled.

Reply:
@philipphofmann any idea?

Reply:
I have no clue. The feature is anyways experimental, and we need to have another look at it. I'm fine with this change, as long as the test runs again. Thanks, @kevinrenskers